libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/SignupBottomSheetDialog.java

```diff
@@ -54,6 +54,7 @@ public void onCancel(DialogInterface dialog) {
         });
 
         setContentView(layout);
+        setTitle(R.string.signup_title);
```

Comment:
We can just use `R.string.sign_up`. “Sign up for WordPress” is incorrect anyway. It's missing the ”.com”. 🙂